style.css

```diff
@@ -1,20 +1,158 @@
-.container {
-  display: block;
-  max-width: 500px;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  background: url(images/index.jpeg) no-repeat center center/ cover;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 60px;
+  border-bottom: 2px solid red;
+  background-color: #fff;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#header-nav {
+  margin-top: 15px;
+  margin-left: 24px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  background-image: linear-gradient(45deg, #4d0418, #eb1d1d, rgb(214, 146, 57), #eb1d1d);
+  color: transparent;
+  -webkit-background-clip: text;
+}
+
+ul {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding-top: 0;
+}
+
+nav li {
+  list-style: none;
+  margin-right: 20px;
+  border-right: 2px solid black;
+  padding-right: 10px;
+}
+
+.contact-me-1 li {
+  margin: 20px;
+}
+
+header nav li:last-child {
+  border: none;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+.active {
+  font-size: 16px;
+  font-weight: 400;
+  color: red;
+}
+
+.time-set {
+  position: absolute;
+  top: 80px;
+  right: 40px;
+}
+
+.time-set p {
+  font-size: 12px;
+  color: #fff;
+  font-weight: 700;
+}
+
+.container .book-list-1 {
+  display: none;
+  width: 500px;
   margin-left: auto;
   margin-right: auto;
+  border-radius: 8px;
+}
+
+.container .add-book-1 {
+  display: none;
+}
+
+h1 {
+  text-align: center;
+  font-size: 24px;
+  font-weight: 800;
+  margin-bottom: 20px;
+}
+
+#book-list h1 {
+  color: #fff;
+}
+
+.container .contact-me-1 {
+  display: none;
+  width: 80%;
+  margin-left: 10%;
+  background-color: #fff;
+  border-radius: 15px;
+}
+
+h2 {
+  color: #fff;
+  text-align: center;
+  font-size: 24px;
+  font-weight: 700;
+  font-style: italic;
+  margin-bottom: 20px;
+}
+
+.contact-me-1 h2 {
+  color: red;
+  padding-top: 20px;
+}
+
+.contact-me-1 p {
+  text-align: start;
+  padding: 10px;
+  font-size: 16px;
+  line-height: 1.5rem;
+  margin-left: 20px;
+}
+
+.contact-me-1 ul {
+  display: block;
+  margin-left: 50px;
+  margin-top: -20px;
+  padding-bottom: 30px;
 }
 
 .library {
   position: relative;
   border: 1px solid black;
   margin-bottom: 20px;
   background-color: white;
+  border-radius: 15px;
 }
 
 .library .new-lib {
-  width: 98%;
-  margin-left: 5px;
+  width: 90%;
+  margin: 5px 20px;
   display: inline-flex;
```

Comment:
[OPTIONAL]
- [ ] You may consider using a list for your book item, and style it accordingly as it is in the wireframe. When we delete an element on your App you can see how the style does not remain consistent.
`hint: You can grab each book in a li and set for example the ones on an even position to #fff color`
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199681538-f12e2591-026e-4bcf-afbe-865a21fe31ae.png)

Template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199682064-6ef06c2a-0fb9-4c25-b39c-b2e7561775c9.png)